dandelion_manuscript/notebook/08_mvTCR/mvTCR_dandelion.ipynb

```diff
@@ -0,0 +1,303 @@
+{
```

Comment:
<p>There’s no plots??</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/zktuong/dandelion-demo-files/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
The code that was used to generate the plots is in this notebook, but my code was not run in any Jupyter notebook. Instead it was executed on the farm using bsub in a loop over the different GEX to VDJ ratios, since it takes ~12h per run and requires GPUs. Thus there is no image of a plot at the end of this example notebook, only the code.

Reply:
Ok thanks!